tardis/plasma/standard_plasmas.py

```diff
@@ -153,3 +166,43 @@ def assemble_plasma(config, model, atom_data=None):
                         property_kwargs=property_kwargs, **kwargs)
 
     return plasma
+
+
+class PlasmaHDFReaderWriter(HDFReaderWriter, BasePlasma):
+
+    @classmethod
+    def from_hdf(cls, file_path, path, model, atom_data_fname=None):
```

Comment:
@wkerzendorf I want to place this `from_hdf` method under `BasePlasma` class , but this function(`from_hdf`) depends on `assemble_plasma` . If I place this function in `BasePlasma` class , it forms a case of `circular imports`. I am confused here , how can I proceed to refactor this??